src/components/profile/profilemenu/ProfileMenu.js

```diff
@@ -0,0 +1,226 @@
+import Text from "@/components/base/text/Text";
```

Comment:
en lille beskrivelse i toppen af nye filer //@file ProfileMenu. js - shows a profile menu for logged in user .. etc. ..